hiding keyboard/ContentView.swift

```diff
@@ -0,0 +1,89 @@
+
+import SwiftUI
+
+//Struct que representa la pantalla principal de la app. Muestra contenido de pantalla
+struct ContentView: View {
+//    Variables que hacen la interfaz se actualice automaticamente. el checkamount es el monto total de la cuenta y el @State dice: Vigila esta variable por si cambia
+    @State private var checkAmount = 0.0
+//
+    @State private var numberOfPeople = 2
+    @State private var tipPercentage = 20
+//    PAra saber si el campo de texto (TextFied) está activo o ono (Si el reclado está abierto=
+    @FocusState private var amountIsFocused: Bool
+//Lista propinas
+    let tipPercentages = [0, 10, 15, 20, 25]
+//numOfPeople inicia en 2, pero se suma 2. (relacionada con el PICKER). Se vuelve Double para hacer calculos decimales
+    var totalPerPerson: Double {
+        let peopleCount = Double(numberOfPeople + 2)
+//        Porcentaje a   decimal
+        let tipSelection = Double(tipPercentage)
+//Caclula cuanto es de propina
+        let tipValue = checkAmount / 100 * tipSelection
+//        Cuenta más propina
+        let grandTotal = checkAmount + tipValue
+//        Se divide entre las personas
+        let amountPerPerson = grandTotal / peopleCount
+
+//        Regresa el resultado final Se usa para mostrarlo en pantalla
+        return amountPerPerson
+    }
+
+//    Se define como se ve la app
+        var body: some View{
```

Comment:
Estilo: 
```suggestion
        var body: some View {
```

Agrega el espacio después de la "w". Sé que es trivial, pero el estilo de código es algo en lo que se estarán fijando mucho.